### ISSUE TITLE
"Application Error" - Changes are not saved

### ISSUE THREAD

**Issue by DarkPassenger86** [HabitRPG/habitica#6438]
Hi. I browsed the help forum but didn't see anything similar to this issue.

I have been receiving an "Application Error" all day. It shows up whenever I try to mark off a task or create a new one or basically change anything at all on Habitica. The error message comes up and says that an error has occurred and that any changes may not have been saved (and they never are saved), and then it says to report it of it keeps happening (which it does).

I am using Google Chrome on Windows 8 on my PC.

Here is my account #: b33fab7a-851b-4e4c-9014-aef63903c19b

Thanks!

**Comment by striglia** [comment]
Same for me. In the browser console, I see:

```
https://habitica.com/api/v2/user/batch-update?_v=115&data=1451591452816 
Failed to load resource: the server responded with a status of 503 (Service Unavailable)
```

**Comment by IstraLouise** [comment]
I just started seeing this as a problem this morning at 8:05 PST.. I get an application error twice on my screen that will not go away, even if I reload the page as it says to do.

User ID: f3d96171-562a-4100-a78e-a3792981d837

I am using Firefox 43.0.2 on Windows 10. I didn't start having the problem until I visited The Sandbox from the Guild Leaders and Challenge Creators link to it.  To be fair, I do have three tabs open, all of which are Habitica.
